lab_Arrays/src/main/java/game/Play.java

```diff
@@ -0,0 +1,49 @@
+package game;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Play {
+    private static final Logger log = LogManager.getLogger(RoundHall.class);
+    private RoundHall hall;
+
+    public Play() {
+        this.hall = new RoundHall();
+        playGame();
+    }
+
+    public void playGame() {
+        while (true) {
+            helpInformation();
+            Dor dor = hall.openDor(hall.getHero().selectDor());
+            if (dor.getMonster() != null) {
```

Comment:
dor.getMonster() != null  - it is bed idea to use null as a part of the logic -
1. At least create methods inside door isMonsterDoor/hasMonster isArtifactDoor/... and use them to check 
2. Build some abstraction on top of a monster and artifact to avoid storing null object inside the class,
but the first option will be ok